### ISSUE TITLE
wrong doc for categorical_hinge loss 

### ISSUE THREAD

**Issue by openyuanmu** [tensorflow/tensorflow#36807]
## Description of issue (what needs changing):

document for tensorflow.keras.losses.categorical_hinge is wrong

### Clear description

```python
@keras_export('keras.losses.categorical_hinge')
def categorical_hinge(y_true, y_pred):
  """Computes the categorical hinge loss between `y_true` and `y_pred`.
  `loss = maximum(neg - pos + 1, 0)`
  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
  Args:
    y_true: The ground truth values. `y_true` values are expected to be -1 or 1.
      If binary (0 or 1) labels are provided they will be converted to -1 or 1.
    y_pred: The predicted values.
  Returns:
    Categorical hinge loss values.
  """
  y_pred = ops.convert_to_tensor_v2(y_pred)
  y_true = math_ops.cast(y_true, y_pred.dtype)
  pos = math_ops.reduce_sum(y_true * y_pred, axis=-1)
  neg = math_ops.reduce_max((1. - y_true) * y_pred, axis=-1)
  return math_ops.maximum(0., neg - pos + 1.)
```

Should be: `neg=maximum((1-y_true)*y_pred)` and `pos=sum(y_true*y_pred)`

**Comment by ayushmankumar7** [comment]
Can i be assigned to this issue?

**Comment by pavithrasv** [comment]
@ayushmankumar7 feel free to send me a PR.

**Comment by ayushmankumar7** [comment]
Sure. Thanks

**Comment by ymodak** [comment]
Closing this issue since the associated PR has been merged. Thanks!
